frontend/src/components/ui/AppFacilityFilter.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-col cols="auto">
+    <AppButton variant="text" :disabled="loading" @click="toggleShowFilter()">
+      Filter by facility
+      <v-icon right>mdi-filter</v-icon>
+    </AppButton>
+  </v-col>
+  <v-col cols="3" class="pb-0">
```

Comment:
The alignment needs tweaking at lower resolutions as the text input becomes quite small. Also might need to tweak ApplicationsHistoryView as well as the input slides below the Applications Summary title.

Reply:
Fixed.